### PR TITLE
 Update pending notifications badge when checking out notifications

### PR DESCRIPTION
**Pull request by alejandraarri**
When the Notifications page is mounted, reset the organization notifications as the user is already checking them.

### Acceptance
1. Go to Your organization/Notifications
2. Send some notifications
3. Go to Notifications page and check that the notifications green icon on the avatar disappears when the page is loaded.

### Related Issue
https://github.com/CartoDB/cartodb/issues/14678

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/pages/Notifications.vue [hunk 1]
```diff
@@ -58,6 +58,9 @@ export default {
     isFetching () {
       return this.$store.state.notifications.isFetching;
     }
+  },
+  mounted: function() {
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/javascripts/new-dashboard/pages/Notifications.vue]: Resolved!

File: lib/assets/test/spec/new-dashboard/unit/specs/store/user/user.spec.js [hunk 2]
```diff
@@ -27,6 +27,22 @@ describe('UserStore', () => {
...
+          icon: 'alert',
+          received_at: '2019-02-19T09:41:29.646Z'
+        }]
+      };
+
+      mutations.setOrganizationNotifications(state, []);
+
+      expect(state).toEqual({
+        organizationNotifications: []
+      })
```
[the remark this answers is not in the document]
Comment (alejandraarri) [reply lib/assets/test/spec/new-dashboard/unit/specs/store/user/user.spec.js]: Resolved!

File: lib/assets/test/spec/new-dashboard/unit/specs/pages/notifications.spec.js [hunk 3]
```diff
@@ -25,6 +25,19 @@ describe('NotificationsPage.vue', () => {
     });
   });
 
+  describe('mounted method', () => {
+    it('should dispatch user/resetOrganizationNotifications action when page is loaded', () => {
```
Comment (VictorVelarde) [inline lib/assets/test/spec/new-dashboard/unit/specs/pages/notifications.spec.js]: Nice tests... 👍

**Comment by csubira** [comment]
Acceptance done following the next steps:

1. Deploy branch in a dedicated instance with an organization (2 users -> owner + additional user).
2. Enable `new-dashboard-feature` flag for both of them
3. Log in as owner -> send organization notification.
4. Read notification. Check that pending notifications badge disappear.
5. Log in as additional user
6. See notifications badge. Read notification. Check that pending notifications badge disappear.

Working great!

**Pull request merged**